rspack.config.ts

```diff
@@ -159,71 +131,76 @@ const localeCatalog: LocaleCatalog = JSON.parse(
-      new RegExp(`locale\\/${locale}\\/.*\\.po$`),
-      new RegExp(`moment\\/locale\\/${language}\\.js$`),
-    ];
-
-    // module test taken from [0] and modified to support testing against
-    // multiple expressions.
-    //
-    // [0] https://github.com/webpack/webpack/blob/7a6a71f1e9349f86833de12a673805621f0fc6f6/lib/optimize/SplitChunksPlugin.js#L309-L320
-    const groupTest: CacheGroupTest = (module, {chunkGraph}) =>
-      localeGroupTests.some(pattern =>
-        pattern.test(module?.nameForCondition?.() ?? '')
-          ? true
-          : chunkGraph.getModuleChunks(module).some(c => c.name && pattern.test(c.name))
-      );
-
-    // We are defining a chunk that combines the django language files with
-    // moment's locales as if you want one, you will want the other.
-    //
-    // In the application code you will still need to import via their module
-    // paths and not the chunk name
-    localeChunkGroups[group] = {
-      chunks: 'async',
-      name: group,
-      test: groupTest,
-      enforce: true,
-    };
-  });
-
-const babelOptions = {...babelConfig, cacheDirectory: true};
-const babelLoaderConfig = {
-  loader: 'babel-loader',
-  options: babelOptions,
-};
+// supportedLocales
+//   // No need to split the english locale out as it will be completely empty and
+//   // is not included in the django layout.html.
+//   .filter(l => l !== 'en')
+//   .forEach(locale => {
+//     const language = localeToLanguage(locale);
+//     const group = `locale/${language}`;
+//
+//     // List of module path tests to group into locale chunks
+//     const localeGroupTests = [
+//       new RegExp(`locale\\/${locale}\\/.*\\.po$`),
+//       new RegExp(`moment\\/locale\\/${language}\\.js$`),
+//     ];
+//
+//     // module test taken from [0] and modified to support testing against
+//     // multiple expressions.
+//     //
+//     // [0] https://github.com/webpack/webpack/blob/7a6a71f1e9349f86833de12a673805621f0fc6f6/lib/optimize/SplitChunksPlugin.js#L309-L320
+//     const groupTest: CacheGroupTest = (module, {chunkGraph}) =>
+//       localeGroupTests.some(pattern =>
+//         pattern.test(module?.nameForCondition?.() ?? '')
+//           ? true
+//           : chunkGraph.getModuleChunks(module).some(c => c.name && pattern.test(c.name))
+//       );
+//
+//     // We are defining a chunk that combines the django language files with
+//     // moment's locales as if you want one, you will want the other.
+//     //
+//     // In the application code you will still need to import via their module
+//     // paths and not the chunk name
+//     localeChunkGroups[group] = {
+//       chunks: 'async',
+//       name: group,
+//       test: groupTest,
+//       enforce: true,
+//     };
+//   });
 
 /**
  * Main Webpack config for Sentry React SPA.
  */
-const appConfig: Configuration = {
+const appConfig: rspack.Configuration = {
+  experiments: {
+    rspackFuture: {
+      disableTransformByDefault: false,
```

Comment:
this is deprecated, you can apply builtins:swc-loader manually to apply SWC transforms to node_modules

Reply:
This is required for the emotion support. The documentation around rspack and emotion is a little bit out of date btw.

Reply:
thanks, we will update the documentation soon